protected/humhub/modules/tour/widgets/Tour.php

```diff
@@ -26,7 +27,7 @@ class Tour extends Widget
     private static function getTypes(): array
```

Comment:
Not sure here. Should that be dynamic using the TourConfig?

Reply:
Thanks! I just forgot to remove this method, it's not used anymore.